settings.gradle.kts

```diff
@@ -40,5 +40,17 @@ include(
   ":biz:obsolete",
 )
 
+project(":base").projectDir = File(rootDir, "gradle/base")
```

Comment:
Changing the base module's path seems can't solve the warning of https://github.com/Goooler/DemoApp/pull/292#issuecomment-1345538629.